src/js/jsx/sections/transform/Position.jsx

```diff
@@ -127,9 +131,11 @@ define(function (require, exports, module) {
                 layers = document.layers.selected,
                 bounds = document.layers.selectedRelativeChildBounds;
 
-            var disabled = this._disabled(document, layers),
-                tops = collection.pluck(bounds, "top"),
-                lefts = collection.pluck(bounds, "left");
+            var disabled = this._disabled(document, layers);
```

Comment:
For stylistic consistency, all these vars should be collapsed into one declaration.